test/index.ts

```diff
@@ -396,4 +398,32 @@ describe('retry-axios', () => {
     }
     assert.fail('Expected to throw');
   });
+
+  it('should parse Retry-After header for retry delay if available', async () => {
+    const scopes = [
+      nock(url).get('/').reply(429, undefined, {'Retry-After': '1'}),
+      nock(url).get('/').reply(200, 'toast'),
+    ];
+    interceptorId = rax.attach();
+    const res = await axios({url});
+    assert.strictEqual(res.data, 'toast');
+    scopes.forEach(s => s.done());
+  });
+
+  it('should not retry if Retry-After value greater than maxRetryAfter', async () => {
+    const scope = nock(url)
+      .get('/')
+      .reply(429, undefined, {'Retry-After': '2'});
+    interceptorId = rax.attach();
+    try {
+      const cfg: rax.RaxConfig = {url, raxConfig: {maxRetryAfter: 1000}};
+      await axios(cfg);
+    } catch (e) {
```

Comment:
I know I may be using this pattern elsewhere in the test, but it's less than ideal.  Would you mind using an `assert.rejects` instead?

Reply:
Done 👍